rwcore/rw/filesystem.hpp

```diff
@@ -3,51 +3,23 @@
 
```

Comment:
This file can be removed as well, I believe.
Or at least replaced by `namespace rwfs = std::filesystem`.